src/ui/views/session/reticle/Reticle.css

```diff
@@ -28,5 +28,9 @@
 }
 
 .Reticle__blue {
-  background: cornflowerblue;
+  background: blue;
+}
+
+.Reticle__yellow {
+  background: yellow;
```

Comment:
Let's use `--usercolor8: #74d12c;` for people. And rather than naming these classes off their colors let's name them based on what they are used for. That way we can change them completely in the theme.